docs/source/user_guide/installation/gnu-linux-mac.rst

```diff
@@ -14,16 +14,26 @@ To use PyBaMM, you must have Python 3.8, 3.9, 3.10, 3.11, or 3.12 installed.
 
    .. code:: bash
 
-      sudo apt update
-      sudo apt install python3
+      apt update \
+      apt install python3
```

Comment:
It has been a while since I checked, but I thought most distros dropped python2. Is it still necessary to specify python3?

Reply:
On older LTS versions, we do need that because both exist – but since they are all mostly old I added a suggestion about the official Python releases (we left a review at the same time)